File: Jack.Model/Model/ColaboradorCrianca.vb
<Serializable()> _
Public Class ColaboradorCrianca

#Region "Construtor"

    Public Sub New()
        Crianca = 0
        Colaborador = 0
    End Sub


#End Region

    Public Overridable Property Codigo As Integer
    Public Overridable Property Colaborador As Integer
    Public Overridable Property Crianca As Integer
    Public Overridable Property Ano As Integer
    Public Overridable Property NumeroSacola As Integer
    Public Overridable Property IsDevolvida As String
    Public Overridable Property NomeCrianca As String
    Public Overridable Property NomeColaborador As String

    Public Overridable Property NumeroIdade As Integer
    Public Overridable Property MedidaIdade As String

    Public Overridable Property Calcado As Integer
    Public Overridable Property Roupa As String

    Public Overridable ReadOnly Property Idade As String
        Get
            If MedidaIdade = "A" Then
                Return NumeroIdade.ToString() + " Anos"
            Else
                Return NumeroIdade.ToString() + " Meses"
            End If
        End Get
    End Property

End Class

File: Jack.Model/Model/ColaboradorCrianca.cs
using System;

namespace Jack.Model
{
    [Serializable()]
    public class ColaboradorCrianca : BaseModel<ColaboradorCrianca>
    {

        #region "Construtor"

        public ColaboradorCrianca()
        {
            Crianca = new Criancas();
            Colaborador = new Colaborador();
        }

        #endregion

        #region "Fields"
        private Colaborador colaborador;
        private Criancas crianca;
        private int ano;
        private int numeroSacola;
        private string isDevolvida;
        private string nomeCrianca;
        private string nomeColaborador;
        private int numeroIdade;
        private string medidaIdade;
        private int calcado;
        private string roupa;

        #endregion

        #region "Properties"

        public virtual Colaborador Colaborador
        {
            get
            {
                return colaborador;
            }
            set
            {
                colaborador = value;
            }
        }

        public virtual Criancas Crianca
        {
            get
            {
                return crianca;
            }
            set
            {
                crianca = value;
            }
        }

        public virtual int Ano
        {
            get
            {
                return ano;
            }
            set
            {
                ano = value;
            }
        }

        public virtual int NumeroSacola
        {
            get
            {
                return numeroSacola;
            }
            set
            {
                numeroSacola = value;
            }
        }

        public virtual string IsDevolvida
        {
            get
            {
                return isDevolvida;
            }
            set
            {
                isDevolvida = value;
            }
        }

        public virtual string NomeCrianca
        {
            get
            {
                return nomeCrianca;
            }
            set
            {
                nomeCrianca = value;
            }
        }

        public virtual string NomeColaborador
        {
            get
            {
                return nomeColaborador;
            }
            set
            {
                nomeColaborador = value;
            }
        }

        public virtual int NumeroIdade
        {
            get
            {
                return numeroIdade;
            }
            set
            {
                numeroIdade = value;
            }
        }

        public virtual string MedidaIdade
        {
            get
            {
                return medidaIdade;
            }
            set
            {
                medidaIdade = value;
            }
        }

        public virtual string Roupa
        {
            get
            {
                return roupa;
            }
            set
            {
                roupa = value;
            }
        }

        public virtual int Calcado
        {
            get
            {
                return calcado;
            }
            set
            {
                calcado = value;
            }
        }

        public virtual string Idade
        {
            get
            {
                if (medidaIdade == "A")
                {
                    return numeroIdade.ToString() + " Anos";
                }
                else {
                    return numeroIdade.ToString() + " Meses";
                }
            }
        }
        
        #endregion

    }
}
